fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/properties/RecordLayerPropertyTest.java

```diff
@@ -32,7 +34,7 @@ void testDefaultPropertyValue() {
         final RecordLayerPropertyStorage properties = RecordLayerPropertyStorage.newBuilder()
                 .build();
         // Assert the prop value is returned as default value false
-        Assertions.assertEquals(false, properties.getPropertyValue(propertyKey), "Unexpected prop value");
+        Assertions.assertEquals(false, Objects.requireNonNullElse(properties.getPropertyValue(propertyKey), false), "Unexpected prop value");
```

Comment:
```suggestion
        Assertions.assertEquals(false, properties.getPropertyValue(propertyKey), "Unexpected prop value");
```

Arguably, the other tests here should also drop their call to `requireNonNullElse`, but I think we want to validate that `false` doesn't somehow get dropped here and return `null`.

Reply:
Used ```requireNonNull``` instead, so a ```null``` value can cause a NPE in these tests, and we can validate that default value won't be dropped.